packages/fetch/src/fetch.ts

```diff
@@ -1,4 +1,4 @@
-import { AbortSignal, createAbortSignal, withLabels, Resource, Operation } from '@effection/core';
+import { AbortSignal, createAbortSignal, withLabels, Operation, Resource } from '@effection/core';
```

Comment:
What does this change accomplish?

Reply:
Take a look the diff of whole PR. I've just reverted my old changes